cypress/e2e/tests/migration/dynamic-report/issues/allIssues/sort.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(["@tier2"], "Issues sort validations", function () {
+    let applicationsList: Array<Analysis> = [];
+    let application: Analysis;
+
+    before("Load data, create Analysis instance and run analysis", function () {
+        login();
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+
+        // Interceptors
+        cy.intercept("POST", "/hub/application*").as("postApplication");
```

Comment:
**nitpick:** This interceptor is not being used anywhere, and the comment is redundant